src/lib/abstractsocialpostcachedatabase.cpp

```diff
@@ -347,7 +347,7 @@ bool AbstractSocialPostCacheDatabase::read()
                 "SELECT identifier, name, body, timestamp "
                 "FROM posts");
     if (!d->accountIdFilter.isEmpty()) {
-        postQueryString += " WHERE identifier IN (" + filteredPostIds.join(',') + ')';
+        postQueryString += " WHERE identifier IN (\"" + filteredPostIds.join("\",\"") + "\")";
     }
```

Comment:
Should line 320 be removed then, so that it's not checking for Ints, but simply calling toString() on all values?